Dockerfile

```diff
@@ -19,15 +19,15 @@ RUN virtualenv .                          && \
     bin/pip install -r requirements.txt . && \
     rm -rf \~
 
-FROM registry.access.redhat.com/ubi8/ubi-minimal as base
+FROM registry.access.redhat.com/ubi9/ubi-minimal as base
 
 USER 0
 
 WORKDIR /opt/app-root
 
 COPY --chown=1001:0 --from=build /opt/app-root /opt/app-root
 
-RUN microdnf install -y python39 libpq procps-ng && \
+RUN microdnf install -y python libpq procps-ng && \
```

Comment:
AFAIK, the package name is `python3` (the same for `python3-devel`). `python` is historically version 2, although in RHEL9 it would be aliased to ver. 3. It would be "safer" to reference it by package name.